### DISCUSSION TITLE
Running locally

### DISCUSSION THREAD

**Discussion by deathmukh** [neuroneural/brainchop#25] [Q&A] [answered]
hi! can i know how to run this project locally once cloned? i need to try a few changes and i have no idea how to run it. will be of much help. thanks!

**Comment by Mmasoud1** [comment, marked as answer]
Hi @deathmukh, and thanks for reaching out.

1- After cloning the repo to your local drive, navigate to the root directory when you are inside the brainchop folder. 

2- Then,  **open the  command line** inside the root directory and execute the following command to start the local HTTP server:
   
If you have Python -2, from the command line, run:

    **python -m SimpleHTTPServer 8020**     
      
 If you have Python -3, from the command line, run:  

        **python -m http.server 8020** 

3- Now you can open the  browser to see brainchop in action by using this address  **http://localhost:8020/** 

I selected 8020 for the above example; if that address is not available on your machine, you can use any address, such as  8000  to  8099 

Let me know please if that works for you.

**Reply by deathmukh** [reply]
You are a life saver! thank you so muchhhh
